lib/sync/index.js

```diff
@@ -253,7 +285,7 @@ module.exports = {
     stop: stop,
     stopAll: stopAll,
     connect: connect,
-    start: start,
+    start: start, //TODO: looks like we don't need to expose this one anymore? init can be used as start
```

Comment:
@wei-lee  Agreed. Given that a call into `sync` or `syncRecords` will start the relevant sync things if not already done, this can be removed.